scripts/generate_rule.js

```diff
@@ -0,0 +1,152 @@
+const fs = require('fs');
+const path = require('path');
+const { program } = require('commander');
+const testJSContent = require('./generate/testJSContent.js');
+const ruleYAMLContent = require('./generate/ruleYAMLContent.js');
+
+const languageInPathToFiles = {
+  go: 'main.go',
+  java: 'main.java',
+  javascript: 'app.js',
+  php: 'index.php',
+  python: 'main.py',
+  ruby: 'main.rb',
+};
+
+function writeRuleStructure(ruleID) {
+  const ruleNameParts = ruleID.split('_');
+  const rootRuleFilePath = path.join(
+    'rules',
+    ruleNameParts[0],
+    ruleNameParts[1],
+  );
+  try {
+    if (!fs.existsSync(rootRuleFilePath)) {
+      const directoryPath = path.dirname(rootRuleFilePath);
+
+      if (!fs.existsSync(rootRuleFilePath)) {
+        fs.mkdirSync(directoryPath, { recursive: true });
+      }
+
+      fs.writeFileSync(rootRuleFilePath, content, 'utf8');
+      console.error(`testdata file written successfully - ${rootRuleFilePath}`);
```

Comment:
I know this is my mistake from my code, but let's correct it here 🙈 

```suggestion
      console.log(`testdata file written successfully - ${rootRuleFilePath}`);
```